### ISSUE TITLE
[BUG] [Linter] False positive with nested {% else %} logic

### ISSUE THREAD

**Issue by joel-coffman** [djlint/djLint#2412]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: macOS 26.5.2
- Python Version: Python 3.14.7
- djLint Version: djlint, version 1.44.2
- template language: Jinja2

## Issue

A false positive is reported for the following Jinja2 template:

```html
<div>
  {% if not thing %}
  <div>Message</div>
  {% else %}
  <div>
  {% for item in items %}
    <div>{{ item }}</div>
  {% else %}
    <div>No items!</div>
  {% endfor %}
  </div>
  {% endif %}
</div>
```

```terminal
$ djlint --profile jinja example.html 

Linting 1/1 files [####################################]


example.html
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
H025 13:0 Tag seems to be an orphan. </div>

Linted 1 file, found 1 error.

```

The nearest I can tell, the issue is the nested `{% else %}` blocks because neither

```html
<div>
  <div>
  {% for item in items %}
    <div>{{ item }}</div>
  {% else %}
    <div>No items!</div>
  {% endfor %}
  </div>
</div>
```

nor

```html
<div>
  {% if not thing %}
  <div>Message</div>
  {% else %}
  <div>
  {% for item in items %}
    <div>{{ item }}</div>
  {% endfor %}
  </div>
  {% endif %}
</div>
```

are flagged with an error.

## How To Reproduce

See above.

## Contents of djlint.toml/.djlintrc/pyproject.toml [tool.djlint]

N/A
